templates/security/index.html

```diff
@@ -332,7 +334,7 @@ <h2 class="p-muted-heading u-align--center">Ubuntu is trusted by</h2>
     <div class="col-7">
       <h2>Find out why the UK Government puts Ubuntu in first place for security</h2>
       <p><abbr title="Communications-Electronics Security Group">CESG</abbr>, the security arm of the UK government rated Ubuntu as the most secure operating system of the 11 they tested.</p>
-      <p><a class="p-button--neutral" href="/blog/ubuntu-scores-highest-in-uk-gov-security-assessment"><span class="p-link--external">Read the <span class="u-off-screen">UK Gov Report Summary </span>case study</span></a></p>
+      <p><a class="p-button--neutral" href="/blog/ubuntu-scores-highest-in-uk-gov-security-assessment"><span>Read the <span class="u-off-screen">UK Gov Report Summary </span>case study</span></a></p>
```

Comment:
Wasn't introduced by this PR, but a little drive by thing you could do is update this button class, `p-button--neutral` is deprecated because it's exactly the same as `p-button`. There's a couple of instances of it on this page, removing them now would mean (very slightly) less to do when it comes to upgrading to Vanilla 3.0
```suggestion
      <p><a class="p-button" href="/blog/ubuntu-scores-highest-in-uk-gov-security-assessment"><span>Read the <span class="u-off-screen">UK Gov Report Summary </span>case study</span></a></p>
```

Reply:
Nice catch, I will keep a look out for them. Is there a list of all the patterns being deprecated?